data/monster/fey/boogy.lua

```diff
@@ -95,6 +95,9 @@ monster.loot = {
 	{id = 677, chance = 3000, maxCount = 3}, -- small enchanted emerald
 	{id = 236, chance = 15000}, -- strong health potion
 	{id = 814, chance = 3200}, -- terra amulet
+	{name = "Fairy Wings", chance = 15000},
+	{name = "Wooden Spellbook", chance = 3440},
+	{name = "Leaf Star", chance = 3440, maxCount = 7},
```

Comment:
Could you please just fix these 3 lines to keep the standard of all other monsters? If the loot is registered by the name, please use all letters with lower case. Thanks for your contribution